idsvr/README.md

```diff
@@ -232,6 +243,17 @@ curity:
 helm upgrade <release-name> curity/idsvr -f myValues.xml
 ``` 
 
+## Enabling the peristed configuration volume 
+
+It is possible to set the `curity.config.persistentConfigVolume.enabled` which will create a `PersistedVolumeClaim` using the defined `StorageClass`, `AccessMode` and `size`. 
```

Comment:
I would tidy up this comment a little:

Title has a typo:
-Enabling the persistent configuration volume

Then, use persistentVolumeClaim and lower case:

- It is possible to set the `curity.config.persistentConfigVolume.enabled` which will create a `PersistentVolumeClaim` using the defined `storageClass`, `accessMode` and `size`.

Then, add a note on expected usage, maybe something like:

- On the initial deployment a persistent volume and persistent volume claim is created. On redeployments you set an `existingClaim` property to reuse the existing claim.